protos/internal/messaging.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+import "protos/common.proto";
+
+option go_package = "github.com/nivista/steady/internal/.gen/protos/messaging";
+
+message Timer {
```

Comment:
What if I want to delete a timer? What does that message look like?

Reply:
that message is defined in services.proto

Reply:
oh ik what you mean in Kafka
that is just null.

Reply:
If i put a proto in for that message, log compaction wouldnt remove the timer

Reply:
Gotcha